build.gradle

```diff
@@ -32,10 +32,9 @@ dependencies {
 	modImplementation ("net.fabricmc:fabric-loader-1.8.9:${project.loader_version}") {
 		exclude module: "guava"
 	}
-	// Fabric API. Most mods will need this, so it's included by default. If you know for a fact you don't, it's not required and can be safely removed.
+	// Most mods will need this, so it's included by default. If you are sure you don't need it, it's not required and can be safely removed.
```

Comment:
I feel that it is important to explicitly say what fapi is because all new users (that aren't familiar with fabric or its toolchain) will start here, with the example mod. Saying most mods will need it but not saying what it actually is is vague and concern

Reply:
Would `Essential hooks and patches for modding with Fabric. Only remove them if you know what you are doing!` be better?

Reply:
Reiterating the name doesn't help much. Calling it "essential" sounds contradictory. I'd just link to the Fabric wiki so they know how to use it.

Reply:
`Fabric API provides hooks for events, item registration, and more. As most mods will need this, it's included by default. If you know for a fact you don't, it's not required and can be safely removed.`
How about this, because it explains what fapi is and how it is not technically required

Reply:
yes, I think this is good